models/cart-discount/src/cart-discount-draft/presets/sample-data-fashion/luxe-spend.ts

```diff
@@ -0,0 +1,41 @@
+import * as CartDiscountValueAbsolute from '@commercetools-test-data/cart-discount-value-absolute';
+import * as CentPrecisionMoney from '@commercetools-test-data/cent-precision-money';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import * as CartDiscountDraft from '../..';
+import { TCartDiscountDraftBuilder } from '../../../types';
+
+const luxeSpend = (): TCartDiscountDraftBuilder =>
+  CartDiscountDraft.presets
+    .empty()
+    .value(
+      CartDiscountValueAbsolute.CartDiscountValueAbsoluteDraft.random()
+        .type('absolute')
+        .money(
+          CentPrecisionMoney.random()
+            .currencyCode('EUR')
+            .centAmount(3000)
+            .fractionDigits(2)
+        )
+    )
+    .cartPredicate(
+      'totalPrice = "500.00 EUR" and customer.customerGroup.key = "luxe"'
```

Comment:
Customer group key should be evaluated similar to the employee sale discount

Reply:
Nice catch, eagle eyes 👁️ 🦅 

da0a5dbbb44c6484ef21ff7554155ada5e2bda52